### PR TITLE
Covid reporter app

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, I did the following:
- Chose [Narrativa API: COVID-19 data] to connect to
- Stored retrieved data from API in redux store
- Showed the list of countries on the home page that could be affected by the covid-19 pandemic.
- Display each Country's data which includes the cities/regions affected.
- Create unit tests for pure functions in your Redux code.
- Create integration tests for my application using the React Testing Library.
- Mock the access to the API, so that my tests don't send actual requests.
- Mock the connection to the Redux Store.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+[View Life](https://duls-covid-reporter.herokuapp.com/)
```
Comment (arslanbisharat) [inline README.md]: - [x] let's add a recorded video as requested in the guidelines

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Make sure you follow the correct gitflow. 
- [x] make sure to add linters
- [x] all the changes are not present in the pull request make sure you merge the previous request properly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+- [View video](https://www.loom.com/share/2f173a6ca1594c60a9e760d6d6ef74a3)
+- [View Demo](https://duls-covid-reporter.herokuapp.com/)
```
Comment (rloterh) [inline README.md]: - [x] OPTIONAL
Your Pull Request is indicating that `README.md` was the only file you worked on for this project. Kindly ensure your project have the new features and modifications of the project code **committed to the repository**.

Comment (rloterh) [inline README.md]: - [x] The styling of the list on your Homepage does not follow the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
image


**The original design:**
image

Comment (rloterh) [inline README.md]: - [x] To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.

Comment (rloterh) [inline README.md]: - [x] As specified, kindly implement a **redux filter/ search** functionality on your Homepage so as to enable filtering of countries.

Comment (rloterh) [inline README.md]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon these aspects of your ReadMe. 
- [x] How to set up, run and test this project should be detailed, clear, and well organized.
- [x] The code aspects in each procedure should be in backticks`.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Duls, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Kindly take your time to improve upon those issues outlined in the code review(s), and please check to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked **[OPTIONAL]** under this review:

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md]: This issue is well resolved! ✔️

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @wuyepabdul,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a Pull Request with unresolved conflicts, kindly resolve your conflicts.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @wuyepabdul,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Pull Request that has no linters check set up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**